x11rb-protocol/src/utils.rs

```diff
@@ -59,20 +51,19 @@ mod raw_fd_container {
         /// This is similar to dropping the `RawFdContainer`, but it allows
         /// the caller to handle errors.
         pub fn close(self) -> Result<(), std::io::Error> {
-            let fd = self.into_raw_fd();
-            nix::unistd::close(fd).map_err(|e| e.into())
+            todo!()
```

Comment:
`rustix::io::close` does not return a `Result`, so there is nothing that can be done here: https://docs.rs/rustix/latest/rustix/io/fn.close.html

I guess that means dropping this function is the only possible thing. "The ecosystem says so" (`std` also has no way to check for close errors).